app/lib/features/onboarding/pages/register_page.dart

```diff
@@ -240,7 +240,7 @@ class _RegisterPageState extends ConsumerState<RegisterPage> {
                                   TextSpan(
                                     recognizer: TapGestureRecognizer()
                                       ..onTap = () {
-                                        debugPrint('Terms of Service"');
+                                        _log.info('Terms of Service"');
```

Comment:
these aren't really meant to be logged but instead replaced with actual content ... the print is there for us to notice ... but whatever.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1495/commits/d67ca486cf571f3de5170bc80b20bbbb5409502c